crowbar_framework/app/barclamps/apache_hadoop_barclamp.rb

```diff
@@ -15,32 +15,36 @@
 # limitations under the License.
 #
 
-barclamp:
-  name: hadoop_infrastructure
-  display: Hadoop Infrastructure
-  version: 0
-  requires:
-    - '@crowbar'
-  member:
-    - apachehadoop
-  supercedes:
-    - hadoop
-    - zookeeper
-    - hive
-    - sqoop
+class ApacheHadoopBarclamp < Crowbar::Registry::Barclamp
+  name "apachehadoop"
+  display "Apache Hadoop"
+  description "Hadoop framework barclamp collection"
 
-crowbar:
-  layout: 1
-  order: 300
-  run_order: 300
-  chef_order: 300
+  member [
+    "apachehadoop"
+  ]
 
-debs:
-  pkgs:
-    # SQL
-    - mysql-server
-    - postgresql
-    - postgresql-server
+  requires [
+    "@crowbar"
+  ]
 
-smoketest:
-  timeout: 900
+  listed false
+
+  layout 1
+  version 0
+  schema 3
+
+  order 200
+
+  nav(
+    barclamps: {
+      apachehadoop: {
+        order: 50,
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.